lib/rules/web/admin_console/4.8/monitoring_alerts.xyaml

```diff
@@ -354,15 +356,16 @@ list_alerts_by_filters_clear:
 list_alerts_by_filters:
   action: click_alert_filter_dropdown
   action: click_alert_filter_item
+  action: click_alert_filter_dropdown
```

Comment:
I can see it exist two line above.

Reply:
> I can see it exist two line above.

One is open drop down, then select an item, the later one is closed drop down list